### PR TITLE
Docker config and AWS credentials for Image building

### PR DESCRIPTION
**Pull request by nickschuch**
https://github.com/GoogleContainerTools/kaniko/blob/master/README.md#pushing-to-amazon-ecr

### CODE REVIEW

File: pkg/utils/aws/credentials/credentials.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package credentials
+
+const (
+	// AccessKeyID used to authentication with AWS.
+	AccessKeyID = "AWS_ACCESS_KEY_ID"
+	// SecretAccessKey used to authentication with AWS.
+	SecretAccessKey = "AWS_SECRET_ACCESS_KEY"
```
Comment (acbramley) [inline pkg/utils/aws/credentials/credentials.go]: We really don't have these anywhere else?

Comment (nickschuch) [reply pkg/utils/aws/credentials/credentials.go]: Nope!

Comment (nickschuch) [reply pkg/utils/aws/credentials/credentials.go]: Follow up PR to update the others if we have them hardcoded. Even looked in the AWS go lib for consts. Why not consts!

File: internal/controller/mysql/image/controller_test.go [hunk 2]
```diff
@@ -42,15 +42,25 @@ func TestReconcile(t *testing.T) {
...
+				Image:  "skpr/mtk-build:latest",
+				CPU:    "100m",
+				Memory: "256Mi",
+			},
+			Docker: mysqlimage.GenerateParamsDocker{
+				ConfigMap: "docker-config",
+			},
+			AWS: mysqlimage.GenerateParamsAWS{
+				Secret:    "aws-credentials",
+				KeyID:     "aws.key.id",
```
Comment (nicksantamaria) [inline internal/controller/mysql/image/controller_test.go]: I'm not a fan of these inconsistent key names. Is this derived from somewhere else or could we make it `aws.key.id` and `aws.key.secret` or something similar?

Comment (nickschuch) [reply internal/controller/mysql/image/controller_test.go]: Thats just test data!

**Pull request merged**